FFXIVClientStructs/FFXIV/Client/System/Framework/Framework.cs

```diff
@@ -1,3 +1,4 @@
+using System.Reflection;
```

Comment:
Was this added by mistake? I don't see a use for it in the other changes in the file.

Reply:
Yeah, my IDE is overzealous at adding things but terrible at removing them after the fact. I'll fix that.